### PR TITLE
OpenStack: Skip in-tree cinder provisioner tests

### PR DESCRIPTION
**Pull request by mandre**
In preparation for kube 1.26 rebase.

### CODE REVIEW

File: test/extended/util/annotate/rules.go
Comment (jsafrane) [inline test/extended/util/annotate/rules.go:93]: There should be a link to a bug to fix the tests.

**Review by soltysh — CHANGES_REQUESTED** [review]
/hold
I've looked into https://github.com/openshift/kubernetes/pull/1432 which is bringing in k8s 1.26 and several random tests I've searched for I still found there being run. So I'd suggest you to compare and see which exact tests should be skipped and only disable those.

**Comment by mandre** [comment]
> /hold I've looked into [openshift/kubernetes#1432](https://github.com/openshift/kubernetes/pull/1432) which is bringing in k8s 1.26 and several random tests I've searched for I still found there being run. So I'd suggest you to compare and see which exact tests should be skipped and only disable those.

Yeah, I've missed https://github.com/kubernetes/kubernetes/pull/113826 which brings back many of the tests that  https://github.com/kubernetes/kubernetes/pull/67782 removed. However we can't really fine-tune the list of tests to skip more than what we're already doing because these tests can't pass until we land the 1.26 rebase.

So in order to not have the openstack job permafailing till the 1.26 rebase, the correct way to go would be to:
- disable all cinder tests (this change)
- merge https://github.com/openshift/kubernetes/pull/1389
- land the 1.26 rebase
- revert this change

Now if I understand @jsafrane's request correctly, he's asking for a jira to track re-enabling the tests which indeed seems like the right thing to do.

I'll create a Jira and update this patch to reference it.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
/hold cancel

File: test/extended/util/annotate/rules.go
Comment (mandre) [reply test/extended/util/annotate/rules.go:94]: Done - https://issues.redhat.com/browse/OCPBUGS-5029.

**Pull request merged**
